website/blog/2024-04-17-release-candidates.md

```diff
@@ -0,0 +1,34 @@
+---
+title: MLflow Release Candidates
+tags: [mlflow]
+slug: release-candidates
+authors: [mlflow-maintainers]
+thumbnail: img/blog/release-candidates.png
+---
+
+# Announcing MLflow Release Candidates
+
+We are excited to announce the implementation of a release candidate process for MLflow!
+The pace of feature development in MLflow is faster now than ever before, and the team has more exciting things planned in the near future. However, ensuring the stability of production systems is of paramount importance to us, and we want to make sure we’re moving fast without breaking things. With that goal in mind, we've decided to introduce a release candidate (RC) process. The RC process allows us to introduce new features and fixes in a controlled environment before they become part of the official release.
```

Comment:
```suggestion
The pace of feature development in MLflow is faster now than ever before and the core maintainer team has even more exciting things planned in the near future! However, with an increased velocity on major feature development comes with a risk of breaking things. As the maintainers of such a widely used project, we are cognizant of the disruptive nature of regressions and we strive to avoid them as much as we can. Aside from new feature development work, our primary goal is in  ensuring the stability of production systems. While we do have the aspirational goal of moving fast(er), we certainly don't want to move fast and break things.  With that goal in mind, we've decided to introduce a release candidate (RC) process. The RC process allows us to introduce new features and fixes in a controlled environment before they become part of the official release.
```